plugins/BEdita/Core/src/Mailer/Preview/UserMailerPreview.php

```diff
@@ -13,11 +13,14 @@
 
 namespace BEdita\Core\Mailer\Preview;
 
+use Cake\Utility\Text;
 use DebugKit\Mailer\MailPreview;
 
 /**
  * Mail preview for UserMailer
```

Comment:
`\BEdita\Core\Mailer\UserMailer` is not used, then I'd rather say "Mail preview for user mailer"